nixos/default.nix

```
@@ -33,6 +42,15 @@ in
         <option>users.users.&lt;name?&gt;.packages</option> option.
       '';
 
+      backupExt = mkOption {
```

Best to avoid unnecessary abbreviations.
```suggestion
      backupFileExtension = mkOption {
```